src/main/java/com/spochi/dto/UserResponseDTO.java

```diff
@@ -37,9 +43,24 @@ public void setNickname(String nickname) {
         this.nickname = nickname;
     }
 
-    public int getAmount_of_initiatives() { return amount_of_initiatives; }
+    public int getAmount_of_initiatives() {
+        return amount_of_initiatives;
+    }
+
+    public void setAmount_of_initiatives(int amount_initiavies) {
+        this.amount_of_initiatives = amount_initiavies;
+    }
+
+    public void setInitiatives_by_status(Map<Integer, Integer> initiatives_by_status) {
 
-    public void setAmount_of_initiatives(int amount_initiavies) {this.amount_of_initiatives = amount_initiavies;}
+        this.initiatives_by_status.put(InitiativeStatus.PENDING.getId(), initiatives_by_status.get(InitiativeStatus.PENDING.getId()));
+        this.initiatives_by_status.put(InitiativeStatus.APPROVED.getId(), initiatives_by_status.get(InitiativeStatus.APPROVED.getId()));
+        this.initiatives_by_status.put(InitiativeStatus.REJECTED.getId(), initiatives_by_status.get(InitiativeStatus.REJECTED.getId()));
+    }
```

Comment:
para mi acá podes hacer un setter comun:
`this.initiatives_by_status = initiatives_by_status;`

Reply:
✅